contracts/zero-ex/contracts/src/transformers/bridges/ArbitrumBridgeAdapter.sol

```diff
@@ -23,19 +23,31 @@ pragma experimental ABIEncoderV2;
 import "./AbstractBridgeAdapter.sol";
 import "./BridgeProtocols.sol";
 import "./mixins/MixinBalancerV2.sol";
+import "./mixins/MixinBalancerV2Batch.sol";
 import "./mixins/MixinCurve.sol";
+import "./mixins/MixinCurveV2.sol";
 import "./mixins/MixinDodoV2.sol";
+import "./mixins/MixinDodo.sol";
+import "./mixins/MixinKyberDmm.sol";
 import "./mixins/MixinGMX.sol";
+import "./mixins/MixinNerve.sol";
 import "./mixins/MixinUniswapV3.sol";
+import "./mixins/MixinUniswapV2.sol";
 import "./mixins/MixinZeroExBridge.sol";
 
 contract ArbitrumBridgeAdapter is
     AbstractBridgeAdapter(42161, "Arbitrum"),
     MixinBalancerV2,
+    MixinBalancerV2Batch,
     MixinCurve,
+    MixinCurveV2,
+    MixinDodo,
```

Comment:
We probably don't need Dodo V1 it's dry. Currently even V2 router isn't working right for us